appserver/admingui/core/src/main/java/org/glassfish/admingui/util/SunOptionUtil.java

```diff
@@ -110,9 +110,11 @@ public static Option getOption(String value, String label) {
         SelectItem[] options =
                 (SelectItem[]) Array.newInstance(SUN_OPTION_CLASS, size);
         options[0] = getSunOption("", "");
-        for (int i = 0; i < size-1; i++) {
-            SelectItem option = getSunOption(values[i], values[i]);
```

Comment:
NPE for `values` would never happen because of indexes and loop condition but sonar can't figure this out so I helped with the extra null check.